fred/fred.py

```diff
@@ -67,9 +67,9 @@ async def on_ready(self):
         self.isReady = True
         self.logger.info(f"We have logged in as {self.user} with prefix {self.command_prefix}")
 
-    @staticmethod
-    async def on_reaction_add(reaction: nextcord.Reaction, user: nextcord.User) -> None:
+    async def on_reaction_add(self, reaction: nextcord.Reaction, user: nextcord.User) -> None:
         if not user.bot and reaction.message.author.bot and reaction.emoji == "❌":
+            self.logger.info("Removing my own message because someone reacted with ❌.")
```

Comment:
Adding the user id as metadata would be nice

Reply:
Fixed in [f71cffa](https://github.com/satisfactorymodding/FICSIT-Fred/pull/145/commits/f71cffaab18df7d61b63c6977e7a35fb3fb99b0c)